tests/src/test/scala/cats/tests/ValidatedTests.scala

```diff
@@ -16,6 +18,22 @@ class ValidatedTests extends CatsSuite {
   checkAll("Validated[String, Int] with Option", TraverseTests[Validated[String,?]].traverse[Int, Int, Int, Int, Option, Option])
   checkAll("Traverse[Validated[String,?]]", SerializableTests.serializable(Traverse[Validated[String,?]]))
 
+  implicit val arbitraryValidated: Arbitrary[Validated[String, Int]] = Arbitrary {
+    for {
+      valid <- arbitrary[Boolean]
+      validated <- if (valid) arbitrary[Int].map(Valid(_))
+                   else arbitrary[String].map(Invalid(_))
+    } yield validated
+  }
+
+  implicit val arbitraryTryInt: Arbitrary[Try[Int]] = Arbitrary {
+    for {
+      success <- arbitrary[Boolean]
+      t <- if (success) arbitrary[Int].map(Success(_))
+      else Gen.const(Failure(new Throwable {}))
```

Comment:
Scalacheck comes with an `Arbitrary[Throwable]`, in case you want to be even more arbitrary.